lib/shared/permissions.js

```diff
@@ -73,6 +73,21 @@ exports.isElevated = () => {
 };
 
 /**
+ * @summary Quote a string
+ * @function
+ * @private
+ *
+ * @param {String} string - input string
+ * @returns {String} quoted string
+ *
+ * @example
+ * const result = doubleQuoteString('foo');
+ */
+const doubleQuoteString = (string) => {
+  return `"${string}"`;
```

Comment:
Hmmmm, I wonder if this should additionally be escaping `"` to `\"` inside the string, in case the string already contains a double-quote?  Quoting and string-escaping always causes headaches ;)